lepton/sdk_example/create-ray-job.py

```diff
@@ -0,0 +1,268 @@
+        mount_list = mount.split(":")
+        if len(mount_list) != 4:
+            raise ValueError(f"Invalid mount format: {mount}. Use PATH:MOUNT_PATH:STORAGE_TYPE:STORAGE_NAME format.")
+        from_obj=f'{mount_list[2]}:{mount_list[3]}' ## storage system in format <storage_type>:<storage_name>
+        mount_objects.append(set_mount(mount_list[0],mount_list[1],from_obj))
+
+    spec = LeptonRayClusterUserSpec(
+        image=image,
+        ray_version=image_version,
+        image_pull_secrets=[image_pull_secret],
+        head_group_spec=RayHeadGroupSpec(
+            resource_shape=head_resource_shape,
+            min_replicas=1,
+            affinity=LeptonResourceAffinity(
+                allowed_dedicated_node_groups=[get_node_group_id(node_group_name, client)]
+            ),
+            envs=env_list,
+            mounts=mount_objects
+        ),
+        worker_group_specs=[RayWorkerGroupSpec(
+            resource_shape=worker_resource_shape,
+            min_replicas=worker_num_replicas,
+            affinity=LeptonResourceAffinity(
+                allowed_dedicated_node_groups=[get_node_group_id(node_group_name, client)]
+            ),
+            envs=env_list,
+            mounts=mount_objects
+        )]
+    )
+
+    lepton_raycluster = LeptonRayCluster(
+            metadata=Metadata(
+                id=name,
+                name=name,
+                visibility=LeptonVisibility.PRIVATE if is_private else LeptonVisibility.PUBLIC
+            ),
+            spec=spec,
+        )
+
+    created = client.raycluster.create(lepton_raycluster)
+    if created:
+        print(f"Ray cluster {name} created")
+        print(f"Ray dashboard URL will be {client.url}/rayclusters/{name}/dashboard")
+    else:
+        print(f"Ray cluster {name} creation failed")
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Create a Ray cluster with configurable node groups and environment variables",
+    )
+    parser.add_argument(
+        "--name",
+        type=str,
+        required=True,
+        help="Name of the Ray cluster"
+    )
+    
+    parser.add_argument(
+        "--node-group-name",
+        type=str,
+        required=True,
+        help="Name of the dedicated node group"
+    )
+    
+    parser.add_argument(
+        "--head-resource-shape",
+        type=str,
+        required=True,
+        help="Resource shape for head node (e.g., 'gpu.8xh200')"
+    )
+    
+    parser.add_argument(
+        "--worker-resource-shape",
+        type=str,
+        required=True,
+        help="Resource shape for workers (e.g., 'gpu.8xh200')"
+    )
+    
+    parser.add_argument(
+        "--image",
+        type=str,
+        default="ray:2.46.0",
+        help="Container image to use (default: ray:2.46.0)"
+    )
+    
+    parser.add_argument(
+        "--image-version",
```

Comment:
Should this be `--ray-version` instead since it's referring to the Ray version? Otherwise without looking at the help almost seems like it could be the image tag?